Controller/Payment/Webhook.php

```diff
@@ -333,13 +382,24 @@ protected function orderPaid(array $post)
 
                 if ($order)
                 {
-                    if (in_array($order->getStatus(), [static::STATUS_PENDING, static::STATUS_PROCESSING]))
+                    if (in_array($order->getStatus(), [static::STATUS_PENDING, static::STATUS_PROCESSING]) or
+                        ($order->getState() === static::STATE_NEW and
+                         $order->getStatus() === static::STATUS_CANCELED)
+                    )
                     {
+                        $this->logger->info("Razorpay Webhook: "
+                        . " Event: " . $post['event']
```

Comment:
Please follow the multiple arguments, alignment formatting rules

Reply:
@nikhilsharma1100 Updated alignment formatting for the multiple arguments